### PR TITLE
Add some more spaceport messages.

### PR DESCRIPTION
**Pull request by petervdmeer**
**Content (New human spaceport messages)**

## Summary
Added some more new human spaceport news messages. I already PRed #5674, so sending in a new PR with some more messages that I came up with. (If both PRs stay in review for a long time, then I will probably combine them.)

## Save File
N/A

## PR Checklist
N/A

### CODE REVIEW

File: data/human/news.txt [hunk 1]
```diff
@@ -674,6 +675,7 @@ news "actor"
 			"Right now I'm mostly just doing commercials, but I'm working on a concept for a zero-gravity musical that will be truly epic."
 			"I have a rich client on Hestia who pays me a thousand credits a day just for recording videos where I bestow extravagant compliments on him."
 			"I recommend that you don't look at any media from the 28th century. They're so laden with pop-culture references that it's painful to watch."
+			"Hey, you are so beautifull, do you want to marry me? Haha, just kidding, I'm rehearsing for my new role."
```
Comment (Terin) [inline data/human/news.txt:678]:
```suggestion
			"Hey, you are so beautiful, do you want to marry me? Haha, just kidding, I'm rehearsing for my new role."
```

File: data/human/news.txt [hunk 2]
```diff
@@ -870,6 +872,7 @@ news "human syndicate manager"
 			"You look like the kind of captain who could benefit from a Syndicate Proton Gun. Would you care to schedule a test-firing?"
 			"Have you considered a Syndicate Freighter for your next ship? Captains like you swear by them!"
 			"Nobody ever got fired for buying Syndicate."
+			"I would love to tell you about my job, but just telling you I work for the Syndicate probably already violates the non-disclosure agremeement I signed."
```
Comment (Terin) [inline data/human/news.txt:875]:
```suggestion
			"I would love to tell you about my job, but just telling you I work for the Syndicate probably already violates the non-disclosure agreement I signed."
```

File: data/human/news.txt [hunk 3]
```diff
@@ -1096,6 +1100,9 @@ news "human pirate"
 			"Look at me like that one more time, and I'll give you plastic surgery free of charge."
 			"Crime pays, and the hours are good!"
 			"I'll retire as a security adviser in the Paradise Worlds. You know the saying: An old poacher makes the best gamekeeper."
+			"Your credits or your life!! Oh you don't have either? Never mind then."
+			"Don't count on the navy to save you if me or any of my friends attack you. Those Navy goons only hang around near the rich planets."
```
Comment (Terin) [inline data/human/news.txt:1104]:
```suggestion
			"Don't count on the Navy to save you if me or any of my friends attack you. Those Navy goons only hang around near the rich planets."
```

**Review by Terin — CHANGES_REQUESTED** [review]

File: data/human/news.txt [hunk 4]
```diff
@@ -674,6 +675,7 @@ news "actor"
 			"Right now I'm mostly just doing commercials, but I'm working on a concept for a zero-gravity musical that will be truly epic."
 			"I have a rich client on Hestia who pays me a thousand credits a day just for recording videos where I bestow extravagant compliments on him."
 			"I recommend that you don't look at any media from the 28th century. They're so laden with pop-culture references that it's painful to watch."
+			"Hey, you are so beautiful, do you want to marry me? Haha, just kidding, I'm rehearsing for my new role."
```
Comment (Amazinite) [inline data/human/news.txt:678]: 👉 👈 
Unless...

File: data/human/news.txt [hunk 5]
```diff
@@ -1096,6 +1100,9 @@ news "human pirate"
 			"Look at me like that one more time, and I'll give you plastic surgery free of charge."
 			"Crime pays, and the hours are good!"
 			"I'll retire as a security adviser in the Paradise Worlds. You know the saying: An old poacher makes the best gamekeeper."
+			"Your credits or your life!! Oh you don't have either? Never mind then."
```
Comment (Amazinite) [inline data/human/news.txt:1103]:
```suggestion
			"Your credits or your life! Oh you don't have either? Never mind then."
```

**Review by Amazinite — CHANGES_REQUESTED** [review]
Stuff like this is small enough that you should just bundle it into an existing PR on the same topic.

**Comment by petervdmeer** [comment]
Will merge the content with #5674

**Pull request closed without merging**